source/DataElement/AMDSBufferGroup.cpp

```diff
@@ -84,9 +84,9 @@ void AMDSBufferGroup::append(AMDSDataHolder *newData, bool elapsedDwellTime)
 		if (specturalCumulativeDataHolder) {
 			AMDSDwellStatusData cumulativeStatusData = specturalCumulativeDataHolder->dwellStatusData();
 
-			emit continuousDataUpdate(specturalCumulativeDataHolder);
-			emit continuousStatusDataUpdate(cumulativeStatusData, count());
-			emit continuousAllDataUpdate(specturalCumulativeDataHolder, cumulativeStatusData, count(), elapsedDwellTime);
+//			emit continuousDataUpdate(specturalCumulativeDataHolder);
```

Comment:
@davidChevrier mentioned there is something wrong with the usage here. I will look into it.